pages/blog/fetch-for-servers.mdx

```diff
@@ -0,0 +1,175 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'JavaScript runs everywhere, so should your servers - here is how',
+  tags: ['fetch', 'whatwg', 'node', 'graphql', 'graphql-yoga'],
+  authors: ['arda'],
+  date: '2022-07-21',
+  description: 'The new way of building servers in JavaScript',
+  image:
+    'https://user-images.githubusercontent.com/727224/104328926-56b0a700-54ba-11eb-9ad3-f6d2b971d55d.jpg',
+  thumbnail:
+    'https://user-images.githubusercontent.com/727224/104328777-29fc8f80-54ba-11eb-87d0-9982af1d8281.jpg'
+}
+
+export default withArticle(meta)
+
```

Comment:
```suggestion
## TLDR
* We made GraphQL Yoga 2.0 **platform agnostic**, so it is able to run anywhere JavaScript can run (Cloudflare Workers, sdfsdf......) thanks to the new **Fetch API standard**
* We've created <ins>Ponyfills</ins> so it will work the same on older versions of Node that don't support the standard
* We've made a **new library** out of it so that any framework and app could achieve the same
* Help other frameworks in the ecosystem to migrate to this new library and standard
```